odl/util/utility.py

```diff
@@ -707,6 +708,15 @@ def signature_string(posargs, optargs, sep=', ', mod=''):
     return part_sep.join(parts)
 
 
+def run_from_ipython():
+    """If the process is run from IPython."""
+    try:
+        __IPYTHON__
+        return True
+    except NameError:
+        return False
```

Comment:
To follow up, according to the discussion [here](https://stackoverflow.com/questions/12693606/reason-for-globals-in-python), `globals()` is (slightly misleadingly) "`locals()` at module level". So my guess was right.